docs/user-guide/custom-themes.md

```diff
@@ -6,12 +6,15 @@ A guide to creating and distributing custom themes.
 
 !!! Note
 
-    If you are looking for third party themes, they are listed in the MkDocs
-    [community wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Themes). If
-    you want to share a theme you create, you should list it on the Wiki.
+If you are looking for third party themes, they are listed in the
+[MkDocs community wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Themes). If 
+you want to share a theme you create, you should list it on the Wiki.
 
-When creating a new theme, you can either follow the steps in this guide to
-create one from scratch or you can download the `mkdocs-basic-theme` as a
+When creating a new theme, you have a few options:
```

Comment:
I'm not sure I see the value in making this a list. I think it was fine as-is, but a list works as well. However, your list's formatting is a [mess](https://travis-ci.org/mkdocs/mkdocs/jobs/163943352#L197). No blank line before the first item, poorly hard wrapped lines (let's keep line length consistent),...